data/sitesEN.json

```diff
@@ -3699,6 +3699,29 @@
     "destination_search_path": "/index.php",
     "destination_content_path": "/"
   },
+  {
+    "id": "en-everythingupgradetree",
+    "origins_label": "Everything Upgrade Tree Fandom Wiki",
+    "origins": [
+      {
+        "origin": "Everything Upgrade Tree Fandom Wiki",
+        "origin_base_url": "everything-upgrade-tree.fandom.com",
+        "origin_content_path": "/wiki/",
+        "origin_main_page": "Everything_Upgrade_Tree_Wiki"
+      }
+    ],
+    "destination": "Everything Upgrade Tree Wiki",
+    "destination_base_url": "eutwiki.com",
+    "destination_platform": "mediawiki",
+    "destination_icon": "everythingupgradetree.png",
+    "destination_main_page": "Main_Page",
```

Comment:
Main page is on the root, so should be just `""`
Alternatively: `"Everything_Upgrade_Tree"`